tests/cases/fourslash/codeFixImportNonExportedMember3.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path='fourslash.ts' />
+// @Filename: /a.ts
+////let a = 1, b = 2, c = 3;
+////let d = 4;
+////export function whatever() {
+////}
+////export { d }

```

Comment:
Would be good to add some test cases where source file has some exports that we cannot use:

```ts
export type { a };
export { b } from "./b";
export * as c from "./c";
// ...
```

Reply:
done